### DISCUSSION TITLE
4.1.0-dev

### DISCUSSION THREAD

**Discussion by Dahkenangnon** [Dahkenangnon/flutter_feathersjs.dart#46] [General]
See https://github.com/Dahkenangnon/flutter_feathersjs.dart/issues/28

See https://github.com/Dahkenangnon/flutter_feathersjs.dart/issues/19#issue-975589167

See https://github.com/Dahkenangnon/flutter_feathersjs.dart/issues/19

<hr /><em>This discussion was created from the release <a href='https://github.com/Dahkenangnon/flutter_feathersjs.dart/releases/tag/4.1.0-dev'>4.1.0-dev</a>.</em>

**Comment by biloo-dev** [comment]
hi @Dahkenangnon Thanks for this wonderful and helpful package ♥♥
i have defrent email password fields name in my database like email => `Cod_user` passsword =>  `MPass`
i have seen your solution for email it's very helpful but the password dos'nt have custom fieldName 
If you could add any information, I would greatly appreciate it

--------------- 
this is image example 
![2023-01-10_215854](https://user-images.githubusercontent.com/37638323/211664058-c499dd25-fb62-4a4a-9a06-683f7331f484.jpg)
